addons/website_sale/security/res_groups.xml

```diff
@@ -11,24 +11,6 @@
     </record>
 
     <!-- Security groups -->
-    <record id="base.group_public" model="res.groups">
-        <field name="implied_ids" eval="[
-            Command.link(ref('account.group_delivery_invoice_address')),
-        ]"/>
-    </record>
-
-    <record id="base.group_portal" model="res.groups">
-        <field name="implied_ids" eval="[
-            Command.link(ref('account.group_delivery_invoice_address')),
-        ]"/>
-    </record>
-
-    <record id="base.group_user" model="res.groups">
-        <field name="implied_ids" eval="[
-            Command.link(ref('account.group_delivery_invoice_address')),
-        ]"/>
-    </record>
-
```

Comment:
IMHO, you could keep this one.
If ecommerce is installed, the feature is enabled by default for backend users.